Networking/Model/NetworkingConfiguration.swift

```diff
@@ -23,35 +23,39 @@ private enum CommunicationProtocol: String {
     It's the only place where these necessary parameters are configured.
  */
 public struct NetworkingConfiguration {
+    /**
+     - Parameters
+     - useSecureConnection: a boolean representing whether the requests
+     will be made using a secure protocol. By default it's enabled.
+     Take into account in case this is disabled, the appropriate
+     exclusions must be added to plist file.
+     - domainURL: the base url the requests will be performed against.
+     - port: the port the requests will be performed against. By default
+     there is no specific port.
+     - subdomainURL: the subdomain url to be appended to domainURL to build
+     the final url (it can be used to specify API versioning). By default it's empty.
+     This url, as a path of the domainURL must start with "/".
+     - usePinningCertificate: a boolean representing if SSL Pinning will be
+     enabled for the performed requests. By default it's disabled. Take into
+     account in case this is enabled, the proper certificate must be included
+     into the application bundle resources.
+     - timeout: the timeout of the requests in seconds. It defaults to 75 seconds.
+     - secondsBetweenPolls: for polling requests, seconds between one polling and the next. It defaults to 1 second.
+     - maximumPollingRetries: Maximum retries until a polling request gives timeout. If it's not set then it will use timeout/secondsBetweenPolls
+     */
```

Comment:
As far as I understand, from posts like [this](http://nshipster.com/swift-documentation/), You must separate it by property like this:
```
/// Boolean representing whether the requests will be made using a secure protocol. By default it's enabled.
/// Take into account in case this is disabled, the appropriate exclusions must be added to plist file.
public var useSecureConnection: Bool = true
```

So that when someone uses your framework, they can see the docs directly in xcode popups.